fixtures/services/bat-safety.yaml

```diff
@@ -1,18 +1,21 @@
 slug: bat-safety
 
-topic: healthcare-and-prevention
-
+topic: healthcare-prevention
+image: curbside-compost
```

Comment:
@LauraTrujillo we need an image placeholder for the data to be able to seed. We'll need to replace this one we have images selected for each content page.